tools/tf_env_collect.sh

```diff
@@ -29,14 +29,29 @@ die() {
 echo "Collecting system information..."
 
 OUTPUT_FILE=tf_env.txt
-python_bin_path=$(which python || which python3 || die "Cannot find Python binary")
+PYTHON_BIN_PATH="$(which python || which python3 || die "Cannot find Python binary")"
+
+HEADER_WIDTH=68
```

Comment:
I just counted the number of "=" symbols in the previous code. There is no reason for this to be 68. If I had been able to choose, I'd have incremented it.